src/actions/EventsActions.ts

```diff
@@ -1,4 +1,4 @@
-import { EventByPKReturnTypes } from 'src/utils/apollo.types';
+import { EventByPKReturnTypes } from './../utils/apollo.types';
```

Comment:
Why not src?

Reply:
changed it a while ago on the right pull